ckanext/validation/plugin/pylons_plugin.py

```diff
@@ -0,0 +1,18 @@
+from ckan import plugins as p
+
+
+class ValidationMixin(p.SingletonPlugin):
+    p.implements(p.IRoutes, inherit=True)
+
+    # IRoutes
+
+    def before_map(self, map_):
+
+        controller = u'ckanext.validation.controller:ValidationController'
+
+        map_.connect(
+            u'validation_read',
```

Comment:
Is there a consistent named route that templates can use to work with either Flask or Pylons without alteration?